tests/test_versioning.py

```diff
@@ -128,7 +128,7 @@ class TestURLReversing(UsingURLPatterns, APITestCase):
     urlpatterns = [
         url(r'^v1/', include(included, namespace='v1')),
         url(r'^another/$', dummy_view, name='another'),
-        url(r'^(?P<version>[^/]+)/another/$', dummy_view, name='another'),
```

Comment:
If I had changed this to:

```python
url(r'^(?P<version>{v1,v2})/another/$', dummy_view, name='another'),
```

Tests would fail:

```
========================================================================== FAILURES ==========================================================================
_____________________________________________________ TestURLReversing.test_reverse_url_path_versioning ______________________________________________________
tests/test_versioning.py:171: in test_reverse_url_path_versioning
    assert response.data == {'url': 'http://testserver/v1/another/'}
E   AssertionError: assert {'url': 'http...ver/another/'} == {'url': 'http:.../v1/another/'}
E     Differing items:
E     {'url': 'http://testserver/another/'} != {'url': 'http://testserver/v1/another/'}
E     Use -v to get the full diff
====================================================== 1 failed, 683 passed, 1 warnings in 5.29 seconds ======================================================

```